static/src/javascripts-legacy/projects/common/modules/crosswords/crossword.js

```diff
@@ -6,7 +6,7 @@ define([
     'lib/mediator',
     'lib/detect',
     'lib/scroller',
-    './anagram-helper/main',
+    'common/modules/crosswords/anagram-helper/main',
```

Comment:
I noticed the relative imports were being returned as undefined (not sure if this was just between es6 and non-es6 modules but it's likely). I don't think there was an explicit complaint from the validators about this though.